settings.gradle.kts

```diff
@@ -0,0 +1,20 @@
+enableFeaturePreview("VERSION_CATALOGS")
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
+
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven(url = "https://maven.pkg.jetbrains.space/public/p/compose/dev" )
+    }
+}
+
+plugins {
+    id("com.dropbox.focus") version "0.4.0"

```

Comment:
Please move the version to `libs.versions.toml` and replace with `libs.versions.plugin.focus`

Reply:
We can't. libs catalog is only available after `settings.gradle.kts` configuration step. 😐